### ISSUE TITLE
Skip link doesn’t show up on iPhone

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#6669]
Tested on iPhone7. To reproduce:

- Activate VoiceOver
- Open Chrome browser
- Swipe right from address bar to move forward through elements.

VoiceOver vocalizes the skipper but it's not visible so double-tap doesn't work. 

This looks like a similar case: https://axesslab.com/skip-links/

**Comment by tobi-or-not-tobi** [comment]
This is related to the fact that the skipper is related to page-slots, and the first skip-link is assigned to the `SiteLogo` component. This component is hidden on mobile, and will therefor break the experience. While we could fix this by assigning an alternative pages-lot, it's not going to be a hack-ish solution, I think we should extend the skip-link to consume page layout (sections) as well, so that we can assign the header. That's the real are we like to focus on anyway for the header skipper. .

**Comment by Xymmer** [comment]
@kacperknapik I tested dev17 with iphone but get the original error. does dev17 have the updated spartacus?

**Comment by Xymmer** [comment]
Miguel clarified that we shouldn't see the skipper, just hear it, works, passing. also miguel tried it and it's working, thanks all
